features/evaluation_spec.rb

```diff
@@ -27,6 +42,7 @@
       class Model1
       end
     RUBY
+    git.checkout(spec_helper)
```

Comment:
Let's commit this changes before we run anything. So, we will be on a safe side forever